src/extension.ts

```diff
@@ -274,20 +274,20 @@ function activateIoTHub(context: vscode.ExtensionContext, outputChannel: vscode.
 
     const azureIoTExplorer = new AzureIoTExplorer(outputChannel, context, iotHubExtTreeDataProvider);
 
-    context.subscriptions.push(iotHubTreeItem);
-    context.subscriptions.push(vscode.window.createTreeView("iotHubExplorer", { treeDataProvider: iotHubExtTreeDataProvider, showCollapseAll: true }));
+    // context.subscriptions.push(iotHubTreeItem);
```

Comment:
instead of comment so much lines here. How about just comment out the call to activateIoTHub?

Reply:
We need `azureIoTExplorer` returned by `activateIoTHub`, so I just comment out part of the code.